blocks/property-listing/cards/cards.css

```diff
@@ -21,10 +22,11 @@
   flex-shrink: 0;
   width: 250px;
   max-width: 90%;
+  background-color: var(--white);
 }
 
 .property-list-cards .listing-tile a {
-  height: 56.25%;
+  height: 75%;
```

Comment:
This throws off the aspect ratio of every other property listing block.

Before: https://main--hsf-commonmoves--hlxsites.hlx.page/drafts/stopp/property-listing
After: https://graybkg--hsf-commonmoves--hlxsites.hlx.page/drafts/stopp/property-listing

Reply:
Understood.  I think that aspect-ratio might work better here.